test/render/test-cases/point-cloud.js

```diff
@@ -49,6 +41,31 @@ const layerProps = {
 };
 
 export default [
+  LASLoader.parseBinary(LAS_BINARY, {skip: 100}).then(lazPointCloud => ({
+    name: 'laz-pointcloud-test',
+    renderingTimes: 1,
+    ...viewProps,
+    viewState: {
+      lookAt: [
+        (lazPointCloud.header.mins[0] + lazPointCloud.header.maxs[0]) / 2,
```

Comment:
Side note: 

I was thinking about separating the concepts of "original header" and "standardized header". 

I.e. each loader would return the original header (where are all field names/value are defined by the format), but also a "standardized" or "normalized" header with a common subset of "metadata" in standardized fields, so that it would be easier to write apps that accept files in multiple formats. 

I think I used an `originalData` or `originalHeader` field in other loaders to store the format specific header, however this part of the design is very open to refinement.